src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int sizeArray = 6;
```

Comment:
let's make this variable a constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names